### PR TITLE
fix(translation): add missing string to translations

### PR DESCRIPTION
**Pull request by FlorianTaut**
Hey guys,

i found a string in `readme.screen.js`
I added it to the translation files but the translators need to translate it now.

Im not sure, probably @housseindjirdeh wanted this string to be not translate or it was just overseen somehow. If its wanted this way please close this pr.

Thank you guys

### CODE REVIEW

**Comment by FlorianTaut** [comment]
So i need you guys to translate this string please:
`repository.readMe.noReadMeFound`

Thank you @siderio2 @Jpfonseca @jouderianjr @machour @lex111 @alejandronanez @RolfKoenders

File: src/locale/languages/es.js [hunk 1]
```diff
@@ -221,6 +221,9 @@ export const erase = {
       deleted: 'DELETED',
       fileRenamed: 'File renamed without any changes',
     },
+    readMe: {
+      noReadMeFound: 'No README.md found',
```
Comment (Antoine38660) [inline src/locale/languages/es.js]: Could you add the `readMeActions` key? It exist in en.js :)

File: src/locale/languages/pt.js [hunk 2]
```diff
@@ -228,6 +228,9 @@ export const pt = {
       deleted: 'REMOVIDO',
       fileRenamed: 'O ficheiro mudou de nome sem qualquer alteração',
     },
+    readMe: {
+      noReadMeFound: 'No README.md found',
```
Comment (Antoine38660) [inline src/locale/languages/pt.js]: Same here about the `readMeActions`

File: src/locale/languages/es.js (hunk 1 above)
Comment (FlorianTaut) [reply src/locale/languages/es.js]: Yeah, the hole `readMe` part was missing in es, not sure why.

**Comment by FlorianTaut** [comment]
Thank you @Antoine38660 now we only need the translations for this string or should we merge without and fix it with following pr's? 👍

File: src/locale/languages/ru.js [hunk 3]
```diff
@@ -229,6 +229,7 @@ export const ru = {
     },
     readMe: {
       readMeActions: 'Действия с README',
+      noReadMeFound: 'No README.md found',
```
Comment (vitalkanev) [inline src/locale/languages/ru.js]: Should be `Не удалось найти README.md`

File: src/locale/languages/fr.js [hunk 4]
```diff
@@ -228,6 +228,7 @@ export const fr = {
     },
     readMe: {
       readMeActions: 'Actions sur le README',
+      noReadMeFound: 'No README.md found',
     },
```
Comment (machour) [inline src/locale/languages/fr.js]: French translation: `Pas de README.md trouvé`

File: src/locale/languages/ru.js (hunk 3 above)
Comment (FlorianTaut) [reply src/locale/languages/ru.js]: Commited! Please check it out 👍

File: src/locale/languages/fr.js (hunk 4 above)
Comment (FlorianTaut) [reply src/locale/languages/fr.js]: Thank you @machour 

Also done, please check it out! 👍

**Comment by chinesedfan** [comment]
@FlorianTaut Thanks for your work! But we may remove this string because of #309.

**Comment by FlorianTaut** [comment]
Hey @chinesedfan, im not sure why this should get in any conflict with #309, could you explain me why please? 👍

**Comment by chinesedfan** [comment]
@FlorianTaut Sorry, I didn't write very clear. I mean, with #309, it is nearly impossible that no ReadMe happens. Because users have no chance to see the entry button first.

**Comment by FlorianTaut** [comment]
@chinesedfan Yeah its possible that this is never shown but we have no string directly written into code, so i think we should do it at this point as well. Dont you think so either?

**Comment by chinesedfan** [comment]
@FlorianTaut If we can confirm, I prefer to remove noReadMe related logic. Then we can skip translating this string. Now we have so many languages to support. 😆  (cc @housseindjirdeh)

**Comment by FlorianTaut** [comment]
@chinesedfan im okay with both, lets wait for @housseindjirdeh‘s opinion and then we can go on with this pr

**Comment by siderio2** [comment]
Should I add the line to the translation or wait??? And if I have to add, how? New PR??

**Comment by FlorianTaut** [comment]
@siderio2 just leave a comment with the translated string please! 👍

**Comment by siderio2** [comment]
This is the Galician translation:
readMe: {
       readMeActions: 'Accións do README ',
       noReadMeFound: 'Non se atopou o README.md',
      },

**Comment by jouderianjr** [comment]
pt-br `noReadMeFound: 'Nenhum README.md foi encontrado'` :dancer:

**Comment by housseindjirdeh** [comment]
Thanks everyone <3

@FlorianTaut what do you think of just leveraging Google Translate for the remainder of the languages and leaving a comment? Comment should allow us to come back to them later just to make sure strings are okay

**Comment by FlorianTaut** [comment]
@housseindjirdeh yes, i will do that today, tried to answer yesterday but gitpoint crashes on every comment 😞

**Comment by housseindjirdeh** [comment]
Thank you @FlorianTaut <3

Are you noticing the constant crashing only on Android? :(

**Pull request merged**